.github/workflows/plugin_update.yml

```diff
@@ -0,0 +1,70 @@
+name: Plugins Update file test
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # run daily at midnight
+  push:
```

Comment:
I prefer using workflow_dispatch instead of push. So you don't need to make a commit to trigger the pipelines.

Please refer to the [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) to learn more.

Reply:
I think we need to remove the push after finishing the cron.

But, if you don't agree, it's also fine.